### PR TITLE
refactor: HTTP bridge into the emqx_bridge_http application

### PR DESCRIPTION
**Pull request by id**
Fixes https://emqx.atlassian.net/browse/EMQX-9568

### <samp>🤖 Generated by Copilot at c3cf619</samp>

Refactor the HTTP bridge and connector features by moving them to a separate application `emqx_bridge_http`, and update the dependencies and references of other applications that use them. Rename the modules `emqx_connector_http` and `emqx_bridge_webhook_schema` to `emqx_bridge_http_connector` and `emqx_bridge_http_schema` respectively, to avoid confusion and reflect the new application structure. Add documentation, testing, and configuration files for the new application.

## PR Checklist
Please convert it to a draft if any of the following conditions are not met. Reviewers may skip over until all the items are checked:

- [ ] Added tests for the changes
- [x] Changed lines covered in coverage report
- [x] Change log has been added to `changes/(ce|ee)/(feat|perf|fix)-<PR-id>.en.md` files
- [x] For internal contributor: there is a jira ticket to track this change
- [x] If there should be document changes, a PR to emqx-docs.git is sent, or a jira ticket is created to follow up
- [x] Schema changes are backward compatible

### CODE REVIEW

File: apps/emqx_bridge_http/docker-ct [hunk 1]
```diff
@@ -0,0 +1 @@
+toxiproxy
```
Comment (thalesmg) [inline apps/emqx_bridge_http/docker-ct:1]: The test suites don't seem to use toxiproxy, so I think this file could be removed.

File: apps/emqx_bridge_http/rebar.config [hunk 2]
```diff
@@ -0,0 +1,10 @@
+%% -*- mode: erlang; -*-
+{erl_opts, [debug_info]}.
+{deps, [ {emqx_connector, {path, "../../apps/emqx_connector"}}
+       , {emqx_resource, {path, "../../apps/emqx_resource"}}
+       , {emqx_bridge, {path, "../../apps/emqx_bridge"}}
+       ]}.
```
Comment (thalesmg) [inline apps/emqx_bridge_http/rebar.config:6]: Strange that this works without declaring `ehttpc` as a dependency.  :thinking:

File: apps/emqx_bridge_http/src/emqx_bridge_http.app.src [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{application, emqx_bridge_http, [
+    {description, "EMQX HTTP Bridge and Connector Application"},
+    {vsn, "0.1.1"},
+    {registered, []},
+    {applications, [kernel, stdlib, emqx_connector, emqx_resource, emqx_bridge]},
```
Comment (thalesmg) [inline apps/emqx_bridge_http/src/emqx_bridge_http.app.src:5]:
```suggestion
    {applications, [kernel, stdlib, emqx_connector, emqx_resource, emqx_bridge, ehttpc]},
```
?

File: apps/emqx_bridge_http/rebar.config (hunk 2 above)
Comment (id) [reply apps/emqx_bridge_http/rebar.config:6]: we have it as a dependency on the top level

**Review by thalesmg — DISMISSED** [review]

**Review by qzhuyan — DISMISSED** [review]

**Review by thalesmg — APPROVED** [review]

**Pull request merged**